internal/command/command.go

```diff
@@ -0,0 +1,29 @@
+package command
```

Comment:
This package name is confusing. I would expect to find the `Command` object in such package.

Reply:
I suggest this lives in `internal/namespaces`.